classpath/javax/net/ssl/SSLServerSocket.java

```diff
@@ -0,0 +1,33 @@
+package javax.net.ssl;
+
+import java.io.*;
+import java.net.*;
```

Comment:
its general better to import the needed classes instead of importing a whole package